load_game.py

```diff
@@ -606,19 +698,19 @@ def load_miihen_start():
 
 
 def load_mrr():
-    FFXC.set_movement(-1, 1)
-    memory.main.wait_frames(30 * 2)
-    FFXC.set_movement(0, 1)
-    memory.main.wait_frames(30 * 1)
-    FFXC.set_movement(1, 1)
-    memory.main.wait_frames(30 * 2)
+    memory.main.await_control()
+    while not pathing.set_movement([-49,166]):
```

Comment:
This one I want to punt on. That whole load file can be heavily cleaned up now, since most of the functions are either nullified or need to be re-written to the new logic.

Reply:
Works for me.